mineinabyss-features/src/main/kotlin/com/mineinabyss/anticheese/AntiCheeseListener.kt

```diff
@@ -45,6 +57,12 @@ class AntiCheeseListener : Listener {
             }
         }
     }
+
+    @EventHandler
+    fun BlockDispenseEvent.preventBackpackPlace() {
+        if (item.type.toString().contains("SHULKER"))
```

Comment:
I'd prefer `"SHULKER" in item.type.toString()`

Reply:
since shulkers are all dyed and the material name isnt just shulker, this seems like the most solid one.
Do check below plox and see if im just smoothbrain or not. Ideally it wouldnt simply cancel but setting the item and breaking it after both were flawed